packages/design-system/src/components/web-components/ds-step-list/ds-step-list.tsx

```diff
@@ -0,0 +1,71 @@
+import { define } from '../preactement/define';
+import { StepListProps } from '../../StepList/StepList';
+import { StepList } from '../../StepList';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'steps',
+  'show-sub-sub-steps',
+  'completed-text',
+  'edit-text',
+  'resume-text',
+  'start-text',
+  'actions-label-text',
+  'substeps-label-text',
+];
+
+interface WrapperProps extends Omit<StepListProps, 'steps' | 'showSubSubSteps'> {
+  steps: string;
+  showSubSubSteps?: string;
+}
+
+const Wrapper = ({
+  steps,
+  showSubSubSteps,
+  completedText,
+  editText,
+  resumeText,
+  startText,
+  actionsLabelText,
+  substepsLabelText,
+  ...otherProps
+}: WrapperProps) => (
+  <StepList
+    steps={typeof steps === 'string' ? JSON.parse(steps) : steps}
+    showSubSubSteps={parseBooleanAttr(showSubSubSteps)}
+    completedText={completedText}
+    editText={editText}
+    resumeText={resumeText}
+    startText={startText}
+    actionsLabelText={actionsLabelText}
+    substepsLabelText={substepsLabelText}
+    {...otherProps}
+  />
+);
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-step-list': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+define('ds-step-list', () => Wrapper, {
+  attributes,
+  events: [
+    [
+      'onStepLinkClick',
```

Comment:
Also there's a way to document this event in storybook. Search for `componentEvents` in the other story files

Reply:
@jack-ryan-nava-pbc, After investigating how StepLink is used in tax-tools and enroll, I think it’s better not to include onStepLinkClick in the ds-step-list api. This way, when a user clicks a link, it will default to standard browser navigation. If we find there’s a need for more nuanced behavior in the future, we can revisit and discuss strategies to support that.